src/web/src/components/SignUp/Forms/GitHubAccount.tsx

```diff
@@ -0,0 +1,191 @@
+      '&:hover': {
+        color: 'black',
+        border: '1px solid #121D59',
+      },
+      '&.Mui-disabled': {
+        backgroundColor: 'inherit',
+      },
+    },
+    avatarPreview: {
+      textAlign: 'center',
+      justifyItems: 'center',
+      alignItems: 'center',
+      justifySelf: 'end',
+      padding: '6%',
+      borderRadius: '5px',
+      [theme.breakpoints.down(600)]: {
+        justifySelf: 'center',
+        padding: '3%',
+        marginTop: '5%',
+      },
+    },
+    username: {
+      fontSize: '1.1em',
+    },
+  })
+);
+
+const gitHubApiUrl = 'https://api.github.com/users';
+
+const GitHubAccount = connect<{}, SignUpForm>((props) => {
+  const classes = useStyles();
+  const { values, setFieldValue } = props.formik;
+  const [validating, setValidating] = useState(false);
+  const [error, setError] = useState('');
+  const controllerRef = useRef<AbortController | null>();
+
+  const validateGit = async () => {
+    if (!values.githubUsername) {
+      setError('');
+      setFieldValue('github', {}, true);
+      return;
+    }
+    try {
+      setValidating(true);
+      if (controllerRef.current) {
+        controllerRef.current.abort();
+      }
+      controllerRef.current = new AbortController();
+      const response = await fetch(`${gitHubApiUrl}/${values.githubUsername}`, {
+        signal: controllerRef.current?.signal,
+      });
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      const res = await response.json();
+
+      setFieldValue('github', {
+        username: res.login,
+        avatarUrl: res.avatar_url,
+      });
+      setError('');
+    } catch (err) {
+      console.error(err, 'Unable to get GitHub profile');
+
+      setError('Unable to get GitHub profile');
+      setFieldValue('github', {}, true);
+    } finally {
+      setValidating(false);
+      controllerRef.current = null;
+    }
+  };
+
+  useEffect(() => {
+    return () => {
+      controllerRef.current?.abort();
+    };
+  }, []);
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.container}>
+        <h1 className={classes.titlePage}>GitHub Account</h1>
+        <h2 className={classes.subtitlePage}>Enter Github username and verify your profile</h2>
+        <div className={classes.infoContainer}>
+          <div className={classes.inputsContainer}>
+            <TextInput
+              label={githubUsername.label}
+              name={githubUsername.name}
+              error={!!error}
+              helperText={error || github.invalidErrorMsg}
+            />
+            <Button className={classes.button} onClick={validateGit} disabled={validating}>
+              Get profile
+            </Button>
+          </div>
+          {!error && (
+            <div className={classes.avatarPreview}>
+              <PostAvatar
+                name={values.github.username || values.displayName}
+                blog={values.github.avatarUrl}
```

Comment:
This is odd, since it's not a blog.  I filed https://github.com/Seneca-CDOT/telescope/issues/2169 to fix it later.